server/routes/openConditions.js

```diff
@@ -30,7 +29,7 @@ module.exports = function Index({ formService, offendersService, userService, au
       const result = await buildFormData(res, req, 'openConditions', form, bookingId, transactionalDbClient)
 
       // Copy offending history charges or skip ?
-      const textExists =
+      const openConditionsExists =
```

Comment:
It's a bit long but what about `openConditionsFurtherChargesTextExists`
or 
`ocFurtherChargesTextExists`?